server/scripts/seed.ts

```diff
@@ -161,6 +162,119 @@ const createDonationForm = (requestGroup = null) => {
     });
 };
 
+const createDonorHomepage = () => {
+    const carouselInterval = 10;
+    const numMapTestimonials = faker.datatype.number({
+        'min': 1,
+        'max': 8
+    });
+    const numStats = 3;
+    const map = {
+        defaultMarkerSize: "53px",
+        markerSizes: ["80px", "75px", "72px", "70px", "65px", "61px", "55px", "53px"],
+        points: [
+            {
+                x: 0.28,
+                y: 0.62
+            },
+            {
+                x: 0.69,
+                y: 0.54
+            },
+            {
+                x: 0.45,
+                y: 0.2
+            },
+            {
+                x: 0.1,
+                y: 0.35
+            },
+            {
+                x: 0.62,
+                y: 0.75
+            },
+            {
+                x: 0.33,
+                y: 0.39
+            },
+            {
+                x: 0.78,
+                y: 0.75
+            },
+            {
+                x: 0.47,
+                y: 0.63
+            }
+        ],
+        testimonials: [],
+        statistics: []
```

Comment:
based on the schema `map` doesn't have a statistics field

Reply:
Good catch! tyvm